ct.sh

```diff
@@ -42,8 +42,13 @@ main() {
         return
     fi
 
-    if [[ "$command" == "lint" ]]; then
+    # Convenience output for other actions to make use of ct config to check if
+    # charts changed.
+    echo "::set-output name=changed::true"
```

Comment:
Would it make sense to declare this in action.yaml as well? https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#outputs - for documentation reasons?

Reply:
Ah I hadn't ever defined an output, but now remember seeing that. Added in a71283b